styles.css

```diff
@@ -46,11 +46,47 @@ nav {
   color: #6070ff;
 }
 
+form ul li {
+  margin: 20px 0;
+}
+
+.techstack div {
+  height: 100%;
+  display: flex;
+  width: 100%;
+}
+
+.menuList {
+  width: 100%;
+  height: 100vh;
+  background: #6070ff;
+  padding: 4rem 2rem;
+  transition: 3s;
+  margin-top: -200vh;
+  position: absolute;
+  opacity: 95%;
+}
+
+.menuList li {
+  line-height: 2rem;
+}
+
+.menuList a {
+  font-weight: 600;
+  font-size: 25px;
+  line-height: 44px;
+  color: #fff;
+}
+
```

Comment:
- [ ] Good job styling your project :100: , but as a project requirement you should stick as much as possible to the Figma template. As you can see below the font size of your elements and the space between them should be bigger :+1: and also the background should be blur.

**Your design**

![Screenshot from 2023-01-30 08-44-56](https://user-images.githubusercontent.com/105117832/215496248-f523ec12-4898-444c-8289-ba9a10a3aa66.png)

**Figma**

![Screenshot from 2023-01-30 08-44-32](https://user-images.githubusercontent.com/105117832/215496407-ab7ee2d0-9d06-4244-b530-887085bcbba9.png)